### DISCUSSION TITLE
Palauan word for SHEET MUSIC

### DISCUSSION THREAD

**Discussion by ChrisPerrette** [tekinged/missing#385] [Q&A]
I am trying to write the phrase "Transcribed by Christopher J. Perrette" in Palauan.
So far, I have, "Ng lulidii el mo er a ongak er a babier a Christopher J. Perrette."
For "sheet music," I have "ongak er a babier." Does this make sense, or would something else be better? Other ideas I had were, "ongak el babier," "babier el ongak," and  "llechukl el ongak."

Any thoughts?

P.S., Sorry if this is the wrong space for this.

**Comment by Gnalu** [comment]
Ng ulidel a Christopher Perrette.
Ng uleiuid er ngii a Christopher Perrette.
Two options for you.

**Comment by JimGeselbracht** [comment]
Probably use the japanese word "hu" which is used for either the melody of a song or sheet music.
    On Tuesday, October 25, 2022 at 11:18:50 AM PDT, Gnalu ***@***.***> wrote:  
 
 

Ng ulidel a Christopher Perrette.
Ng uleiuid er ngii a Christopher Perrette.
Two options for you.

**Comment by ChrisPerrette** [comment]
@JimGeselbracht So something like, “Ng lulidii el mo er a hu a Christopher J. Perrette”?

**Comment by JimGeselbracht** [comment]
Maybe just "ng lulidii a hu a Christopher"?  Since hu is the melody or the written melody.  Mora hu doesn't sound right to me
    On Tuesday, October 25, 2022 at 01:21:22 PM PDT, Chris Perrette ***@***.***> wrote:  
 
 


@JimGeselbracht So something like, “Ng lulidii el mo er a hu a Christopher J. Perrette”?

**Reply by ChrisPerrette** [reply]
I am trying to get communicate the idea that I took the music as sung in the classic recording of "Belau Rekid" and converted/translated it to sheet music. The English "transcribe" covers this whole meaning. Would the Palauan word "oliuid" cover this?

**Comment by Gnalu** [comment]
https://m.youtube.com/watch?v=JA0IF3O_Nzg
 You mean like this!…

**Comment by Gnalu** [comment]
A Chris a Mla oidii Tia el chelitakl (title). Like so…

**Comment by Gnalu** [comment]
I think it has already been done…

**Reply by ChrisPerrette** [reply]
> I think it has already been done…

@Gnalu  Where have you seen it?

**Comment by Lukes-Imeyuns** [comment]
It's hard to translate "sheet music" and "Hu" is used more for the tune or
melody. One would say, "Ak medengei a tekoi er ngii e ng di ak mla obes ra
hu er tial chelitakl." (I know the lyrics but I don't remember the tune.)

IMO, if this note is going to be on THE page, table of contents, Word or
PDF or footnote etc where it's clear what it's referring to then using"
Ulidii a Christopher J. Perrette" ought to be good. Or "Ng Ulidii a..." or
"Tiang a lulidii a..."

I just don't think it can start with "Lulidii" without a preceding word. It
sounds LULIDII when we say it but it's actually "Tiang al Ulidii a..." or
maybe it's "Tiang a lulidii" (I forget the rules) It's just that the "l"
only appears when there's a preceding word. This happens with many words as
you've seen, such as below:

Tial bilcherar = Tiang al bilcherar
Tial dilrur = Tiang al dilrur
Tial tilkir = Tiang al tilkur

Hope that helps!

And by the way, it's awesome that you're doing that. Music should be part
of the Palau school curriculum so kids can learn music notes.

Lukes

**Reply by ChrisPerrette** [reply]
Thanks, I think I will use that. I have a previous iteration of the sheet music already uploaded to tekinged, but I am trying to update it so it looks better.

**Comment by ChrisPerrette** [comment]
Here is the version I have been updating.

[Belau_Rekid.pdf](https://github.com/tekinged/missing/files/9865591/Belau_Rekid.pdf)

**Comment by ChrisPerrette** [comment]
@Lukes-Imeyuns Was this the word you were referring to?
https://www.nihongomaster.com/japanese/dictionary/word/22721/kyoku-%E6%9B%B2-%E3%81%8D%E3%82%87%E3%81%8F
